src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Fixtures/ReleaseRedirectGeneratorExtensions.cs

```diff
@@ -0,0 +1,39 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+using System;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Fixtures;
+
+public static class ReleaseRedirectGeneratorExtensions
+{
+    public static Generator<ReleaseRedirect> DefaultReleaseRedirect(this DataFixture fixture)
+        => fixture.Generator<ReleaseRedirect>().WithDefaults();
+
+    public static Generator<ReleaseRedirect> WithDefaults(this Generator<ReleaseRedirect> generator)
+        => generator.ForInstance(d => d.SetDefaults());
+
+    public static InstanceSetters<ReleaseRedirect> SetDefaults(this InstanceSetters<ReleaseRedirect> setters)
+        => setters
+            .SetDefault(p => p.Slug)
+            .SetDefault(p => p.Created);
+
+    public static Generator<ReleaseRedirect> WithRelease(
+        this Generator<ReleaseRedirect> generator,
+        Release release)
+        => generator.ForInstance(s => s.SetRelease(release));
+
+    public static Generator<ReleaseRedirect> WithReleaseId(
+        this Generator<ReleaseRedirect> generator,
+        Guid releaseId)
+        => generator.ForInstance(s => s.SetReleaseId(releaseId));
+
+    public static InstanceSetters<ReleaseRedirect> SetRelease(
+        this InstanceSetters<ReleaseRedirect> setters,
+        Release release)
+        => setters.Set(rr => rr.Release, release)
+            .SetReleaseId(release.Id);
+
+    public static InstanceSetters<ReleaseRedirect> SetReleaseId(
+        this InstanceSetters<ReleaseRedirect> setters,
+        Guid releaseId)
+        => setters.Set(rr => rr.ReleaseId, releaseId);
```

Comment:
Could also add `WithSlug` / `SetSlug` for completeness.

Same comment also applies to `PublicationRedirectGeneratorExtensions` and `MethodologyRedirectGeneratorExtensions`.

Reply:
good catch, I'll add those in :)